bb_equipment_transforms:
  world:
    frame_id: world
    child_frame_id: map
    translation:
      x: -1.25
      y: -0.6
      z: 0.77
    rotation:
      r: 0.0
      p: 0.0
      y: 0.78
  scan_stationary:
    frame_id: world
    child_frame_id: scan_stationary
    translation:
      x: 0.0
      y: 0.25 
      z: 0.0
    rotation:
      r: 0.0
      p: 0.0
      y: 0.0
  kinect2_stationary:
    frame_id: world
    child_frame_id: kinect2_link
    translation:
      x: -1.7
      y: 0.0 
      z: 1.73
    rotation:
      r: -1.7886
      p: 0.0
      y: -1.04
